ONSdigital/ras-frontstage
# adding functionality to reject 0 byte files

# Motivation and Context
to reject 0 byte files

# What has changed
added functions to check upload file size
added a test to test the upload of 0 byte files

# How to test?
spin up frontstage
get a survey through rops
upload a 0 byte file
- it should reject the file and there should be an option to browse for a new file
